src/store/jetton-store/useJettonStore.ts

```diff
@@ -124,7 +124,14 @@ function useJettonStore() {
       });
     } catch (error) {
       if (error instanceof Error) {
-        showNotification(error.message, "error");
+        const errorRegexes = [/exit-code: 11/, /exit-code: 32/];
```

Comment:
isn't it supposed to be `exit_code`?

in any case you can do it with a single regex: `/exit_code: (11|32)/g`